pkg/sti/errors/errors.go

```diff
@@ -100,7 +111,7 @@ func NewBuildError(name string, err error) error {
 		Message:    fmt.Sprintf("building %s failed", name),
 		Details:    err,
 		ErrorCode:  ErrBuild,
-		Suggestion: "check the assemble script for errors",
+		Suggestion: "check the build output for errors",
```

Comment:
That's already happening, see https://github.com/openshift/source-to-image/blob/master/cmd/sti/main.go#L189-L191